### DISCUSSION TITLE
Simple Creation of "Standard Csv" (right formatted) from Excel worksheets

### DISCUSSION THREAD

**Discussion by manuel-sperrle** [dfinke/ImportExcel#1290] [Show and tell]
Creation of "standard csv" files gets real ugly using regular Excel UI
e.g. : German User -> Excels Delimiter of choice = ";"

Import-Excel made life much easier there - a simple

Import-Excel -Path C:\Excel.xlsx | ConvertTo-Csv -NoTypeInformation

![image](https://user-images.githubusercontent.com/112897804/197325910-de571575-6322-4994-86f8-468beef04ad7.png)
![image](https://user-images.githubusercontent.com/112897804/197325937-bed10954-d867-40c7-8c29-205b1bee792c.png)

**Comment by ninmonkey** [comment]
Are you asking for the docs on the style?

- https://epplussoftware.com/docs/5.7/api/OfficeOpenXml.Style.ExcelHorizontalAlignment.html

**Reply by dfinke** [reply]
@ninmonkey  I believe @SomehowSaltedNameSpace is saying that the PowerShell module is easier and better to use than the Excel UI
